manifests/config.pp

```diff
@@ -69,12 +72,26 @@
     }
   }
   ['PREROUTING', 'INPUT', 'OUTPUT', 'POSTROUTING'].each |$nat_chain| {
+    if versioncmp($kver, '3.17.0') >= 0 {
+      # supports both nat INPUT chain and ip6table_nat
+      $domains = $ferm::ip_versions
+    } elsif versioncmp($kver, '2.6.35') >= 0 {
+      # supports nat INPUT chain, but not ip6table_nat
+      unless ('ip6' in $ferm::ip_versions and 'ip' in $ferm::ip_versions) { next() }
```

Comment:
This isn't going to change within the loop so the entire loop could be skipped, right?

Reply:
indeed, great catch - fixed.